flows/data_transformation/dataflow_ui_plugin/flow.py

```diff
@@ -19,7 +19,7 @@
 
 
 @flow(log_prints=True)
-def dataflow_ui_plugin(json_graph, options):
+def dataflow_ui_plugin(json_graph, importlibs, variables, options):
```

Comment:
import_libs is provided in snake case: https://github.com/OHDSI/d2e/pull/459/files#diff-78d65316cc9e6bf33f80863d04f0527697cfda406c47350c98a8516b47947f5bR293